vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -76,6 +79,30 @@ You will need to restart VS Code after these changes. If PowerShell is still not
                             windowsFullCommand = windowsFullCommand.replace('powershell.exe', powershellReference);
                         }
 
+                        // The install script can leave behind a directory in an invalid install state. Make sure the executable is present at the very least.
+                        if (await this.fileUtilities.exists(installContext.installDir))
+                        {
+                            const dotnetPath = path.join(installContext.installDir, getDotnetExecutable());
+                            if (await this.fileUtilities.exists(dotnetPath))
+                            {
+                                const validator = new DotnetConditionValidator(this.workerContext, this.utilityContext);

```

Comment:
I'm a bit concerned this could throw, and we might want to catch that.

Reply:
I double checked, and this code should never throw, but I understand the concern and good to check!